Allow qt5-webengine to be built on FreeBSD with clang

--- tools/qmake/mkspecs/features/functions.prf.orig	2017-04-11 14:08:45 UTC
+++ tools/qmake/mkspecs/features/functions.prf
@@ -1,5 +1,5 @@
 defineTest(isPlatformSupported) {
-  linux {
+  unix {
     !gcc:!clang {
       skipBuild("Qt WebEngine on Linux requires clang or GCC.")
       return(false)
